api/v1/server/oas/transformers/workflow.go

```diff
@@ -148,7 +148,7 @@ func ToWorkflowVersionConcurrency(concurrency *WorkflowConcurrency) *gen.Workflo
 
 	res := &gen.WorkflowConcurrency{
 		MaxRuns:             concurrency.MaxRuns.Int32,
-		LimitStrategy:       gen.WorkflowConcurrencyLimitStrategy(concurrency.LimitStrategy.ConcurrencyLimitStrategy),
+		LimitStrategy:       gen.ConcurrencyLimitStrategy(concurrency.LimitStrategy.ConcurrencyLimitStrategy),
```

Comment:
is this change related?

Reply:
Yes, somehow changing the package for bundling changed our generated API as well. Seems to not be an issue but will double check on staging.